build.sbt

```diff
@@ -17,10 +17,16 @@ val akkaV = "2.4.11" // Note: akka-http branches from akkaV after 2.4.11
 libraryDependencies ++= Seq(
   "com.github.kxbmap" %% "configs" % "0.4.2",
   "com.typesafe" % "config" % "1.3.0",
-  "org.typelevel" %% "cats" % "0.7.2",
+  "org.typelevel" %% "cats" % "0.7.2"
+    exclude("org.typelevel", "cats-laws_2.11")
+    exclude("org.typelevel", "cats-kernel-laws_2.11"),
   "com.typesafe.akka" %% "akka-actor" % akkaV,
   "com.typesafe.akka" %% "akka-http-experimental" % akkaV,
   "com.typesafe.akka" %% "akka-http-spray-json-experimental" % akkaV,
-  "org.scalatest" %% "scalatest" % "2.2.6" % Test,
-  "com.github.pathikrit" %% "better-files" % "2.13.0"
+  "com.github.pathikrit" %% "better-files" % "2.13.0",
+  //---------- Test libraries -------------------//
+  "org.scalatest" %% "scalatest" % "3.0.1" % Test,
+  "org.pegdown" % "pegdown" % "1.6.0" % Test
 )
+
+testOptions in Test += Tests.Argument(TestFrameworks.ScalaTest, "-oDSI", "-h", "target/test-reports")
```

Comment:
OCD answer: It's syncing up the reporting to be the same as cromwell.

Technical answer via [docs](http://www.scalatest.org/user_guide/using_the_runner):
- `-oD` - show all durations
- `-oS` - show short stack traces
- `-oI` - after each sbt project finishes, of which wdl4s only has one, reprint a summary of all the failed tests
- `-h target/test-reports` - create an html report of the test results